playbooks/roles/mysql/files/edxapp.sql

```diff
@@ -88,6 +88,7 @@ VALUES
   We will set is_active = False. So we must use reset password inorder to set password and to activate the account.
 */
 
+DELETE FROM auth_userprofile where name='oxamaster';
 INSERT into auth_user
 (
   password,
```

Comment:
Oxa-Tools repo is public one, not private one, its security issue, we insert the row in the auth_user with password - SHA1('edx'), though it will ask you change the password once you login but, there is chance of being used by one before  we change the password. 

there is another way to create the superuser using django shell

sudo -u www-data /edx/bin/python.edxapp ./manage.py lms --settings aws create_user -s -p $defaultPwd -e $defaultSuperUserEmail
echo "from django.contrib.auth.models import User; me = User.objects.get(username='"$defaultSuperUserName"'); me.is_superuser=True; me.is_active=False; me.save()" | sudo -u www-data /edx/bin/python.edxapp ./manage.py lms --settings aws shell